resources/dicts/relationship_events/normal_interactions/platonic/increase.json

```diff
@@ -45,7 +45,6 @@
 			"m_c thinks r_c has interesting things to talk about.",
 			"m_c is fascinated by one of r_c's stories.",
 			"m_c likes the way r_c laughs.",
-			"m_c thinks that r_c is good at leading others.",

```

Comment:
Removed

Reply:
thought it was a little vague, and it'd be a little funny for someone to think a 2 month old kit is good at leading others LOL